w3/.babelrc

```diff
@@ -11,7 +11,8 @@
       }
     ],
     // This tells babel we want to use react w/ es6
+    "es2015",
```

Comment:
We shouldn't need this if we already have babel-preset-env

Reply:
👍

Reply:
That's a good point